src/main/groovy/util/StockMovementStatusHelper.groovy

```diff
@@ -0,0 +1,83 @@
+package util
+
+import org.pih.warehouse.api.StockMovementDirection
+import org.pih.warehouse.api.StockMovementType
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.order.Order
+import org.pih.warehouse.order.OrderStatus
+import org.pih.warehouse.requisition.RequisitionStatus
+import org.pih.warehouse.shipping.Shipment
+import org.pih.warehouse.shipping.ShipmentStatusCode
+
+class StockMovementStatusHelper {
+
+    Order order
+
+    Shipment shipment
+
+    StockMovementType stockMovementType
+
+    Location origin
+
+    Location destination
+
+    RequisitionStatus defaultStatus
+
+
+    StockMovementDirection getStockMovementDirection(Location currentLocation) {
+        if (currentLocation == origin) {
+            return StockMovementDirection.OUTBOUND
+        }
+        if (currentLocation == destination || origin?.isSupplier()) {
+            return StockMovementDirection.INBOUND
+        }
+        return null
+    }
+
+    boolean isInbound() {
+        Location currentLocation = AuthService.currentLocation
+        return getStockMovementDirection(currentLocation) == StockMovementDirection.INBOUND
```

Comment:
Basically, if the `currentLocation` is used only once, I don't see a need to create a new variable for that :thinking: 
```suggestion
        return getStockMovementDirection(AuthService.currentLocation) == StockMovementDirection.INBOUND
```

Reply:
For me it looked cleaner to keep it in a separate variable. It might not make a sense from the „memory” POV, but readability is also important. 
If you guys prefer not creating a variable for that, then I’ll change it.